scripts/import/dbSNP_v2/update_new_sets.pl

```diff
@@ -0,0 +1,304 @@
+#!/usr/bin/env perl
+use strict;
+use DBI;
+use Bio::EnsEMBL::Registry;
+use Getopt::Long;
+use Cwd qw(cwd);
+
+
+my ($reg_file, $old_reg_file, $release, $tmp, $help);
+
+GetOptions ("registry=s"        => \$reg_file,
+            "old_registry=s"    => \$old_reg_file,
+            "release=s"         =>  \$release,
+            "tmp=s"             =>  \$tmp,
+            "help|h"           =>   \$help,
+);
+
+# to define temporary directory if it is not defined 
+if (!defined $tmp){
+  $tmp = cwd;
+}
+usage() unless defined $reg_file && defined $release && defined $tmp;
+
+my $registry = 'Bio::EnsEMBL::Registry';
+$registry->load_all($reg_file);
+
+my $db_adaptor = $registry->get_DBAdaptor("homo_sapiens", "variation");
+my $dbh = $db_adaptor->dbc;
+
+my $TMP_DIR = $tmp;
+my $TMP_FILE = "variation_feature.txt";
+my $tmp_vset = "variation_name_set.txt";
+my $tmp_merged = "new_variation_set.txt";
```

Comment:
It could be named "new_variation_set_variation.txt"
variation_set is a different table